### PR TITLE
Adds more shapeshift shapes

### PR DESCRIPTION
**Pull request by ghost**
# Lord Gondolamort has risen

:cl: More Robust Than You
add: Wizards with shapeshift can now transform into cats, gondolas, crabs, or butterflies too!
/:cl:

### CODE REVIEW

**Comment by Armhulen** [comment]
@KorPhaeron haha r-rare space r-ruin

**Comment by RogueSteampunker** [comment]
Because Mindswapping people into Gondolas so they're killed for their pelts has never been so tempting before.

File: code/modules/spells/spell_types/shapeshift.dm [hunk 1]
```diff
@@ -18,7 +18,11 @@
 		/mob/living/simple_animal/pet/dog/corgi,\
 		/mob/living/simple_animal/hostile/carp/ranged/chaos,\
 		/mob/living/simple_animal/bot/ed209,\
-		/mob/living/simple_animal/hostile/construct/armored)
+		/mob/living/simple_animal/hostile/construct/armored,\
+		/mob/living/simple_animal/pet/cat,\
+		/mob/living/simple_animal/butterfly,\
+		/mob/living/simple_animal/pet/gondola,\
+		/mob/living/simple_animal/crab)
```
Comment (KorPhaeron) [inline code/modules/spells/spell_types/shapeshift.dm]: Basically all of these are just suicide for the wizard and the gondola shouldn't be so common on top of that

Comment (MoreRobustThanYou) [reply code/modules/spells/spell_types/shapeshift.dm]: They are basically meant for stealth

**Comment by YakumoChen** [comment]
please don't flood shapeshift with useless options, we already have the corgi for the meme shapeshift.

**Comment by jughu** [comment]
Gimme a gorilla shapeshift now people will fear my true form

File: code/modules/spells/spell_types/shapeshift.dm (hunk 1 above)
Comment (jughu) [reply code/modules/spells/spell_types/shapeshift.dm]: Want stealth? Go mouse

**Comment by ExcessiveUseOfCobblestone** [comment]
just replace all of the ones you added with the nurse and that should be interesting lol.

**Pull request closed without merging**